docs/webhook-config.md

```diff
@@ -114,6 +116,9 @@ Possible parameters are:
 * `order_type`
 * `current_rate`
 * `enter_tag`
+* `gain`
+* `profit_amount`
+* `profit_ratio`
```

Comment:
Do we add gain, profit_amount to everything here since we can long/short ?

Reply:
it's unclear to me what the difference between "gain" and "profit_amount" is ... in my understanding - both specify "i made 10$ profit" ... ?

Reply:
"everything" ... it should be added where it's really available - it will not make sense for order cancel's (for example).

Reply:
Its copied from sell fill. 
`gain = "profit" if profit_ratio > 0 else "loss"
`
I think i need to update this 
`freqtrade/freqtradebot.py`
`def _notify_exit`
to add leverage and is_short. 

Guess i need to update to account for these
RPCMessageType.SHORT_FILL
RPCMessageType.SHORT
RPCMessageType.SHORT_CANCEL

Reply:
I will update the pr to account for those, this is just draft for discussion :+1: